packages/frontend/src/pages/InferenceServerDetails.svelte

```diff
@@ -218,84 +214,124 @@ function handleOnChange(): void {
+              <div>
+                <!-- title -->
+                <div class="flex flex-row">
+                  <span class="text-base grow text-[var(--pd-content-card-text)]">Inference Server</span>
                 </div>
-              </div>
 
-              <!-- server details -->
-              <div class="bg-[var(--pd-content-card-bg)] rounded-md w-full px-4 pt-2 pb-4 mt-2">
-                <span class="text-sm text-[var(--pd-content-card-text)]">Server</span>
-                <div class="flex flex-row gap-4">
+                <!-- inference server details content -->
+                <div class="bg-[var(--pd-content-card-bg)] rounded-md w-full px-4 pt-3 pb-4 mt-2 flex flex-col gap-y-4">
+                  <!-- endpoint URL -->
                   {#if service.status === 'running'}
-                    {#if 'docs' in service.labels}
-                      <Tooltip tip="Open swagger documentation">
-                        <button
-                          on:click={handleOnClick}
-                          class="bg-[var(--pd-label-bg)] text-[var(--pd-label-text)] rounded-md p-2 flex flex-row w-min h-min text-xs text-nowrap items-center underline">
-                          {service.labels['docs']}
-                          <Fa class="ml-2" icon={faBook} />
-                        </button>
-                      </Tooltip>
-                    {/if}
+                    <div class="flex flex-col gap-y-2">
+                      <span class="text-sm text-[var(--pd-content-card-text)]">Inference Endpoint URL</span>
+                      <div class="flex items-center gap-x-4">
+                        <!-- API URL -->
+                        {#if 'api' in service.labels}
+                          <CopyButton
+                            top
+                            class="bg-[var(--pd-label-bg)] text-[var(--pd-label-text)] rounded-md p-2 flex flex-row w-min h-min text-xs text-nowrap items-center"
+                            content={service.labels['api']}>
+                            {service.labels['api']}
+                            <Fa class="ml-2" icon={faCopy} />
+                          </CopyButton>
+                        {/if}
 
-                    {#if 'api' in service.labels}
-                      <CopyButton
-                        class="bg-[var(--pd-label-bg)] text-[var(--pd-label-text)] rounded-md p-2 flex flex-row w-min h-min text-xs text-nowrap items-center"
-                        content={service.labels['api']}>
-                        {service.labels['api']}
-                        <Fa class="ml-2" icon={faPlug} />
-                      </CopyButton>
-                    {/if}
-                  {/if}
+                        <!-- Documentation URL -->
+                        <div class="grow">
+                          {#if 'docs' in service.labels}
+                            <Tooltip tip="Open swagger documentation">
+                              Access
```

Comment:
Access should reflect light/dark theme 


https://github.com/user-attachments/assets/3496a4f6-20ba-46df-bf7d-c937a508e9cb

Also I would say that the tooltip should be visible only when you hover the link `swagger documentation` and not the text `Access`

Reply:
![image](https://github.com/user-attachments/assets/6c5e3e5c-4502-4350-a0f7-c85e27c5457e)

![image](https://github.com/user-attachments/assets/abf81273-822b-4789-a314-a5e2058b8282)

Nice catch! Fixed :heavy_check_mark: